src/inngest/functions/extractPdf/convertToHtml.ts

```diff
@@ -2,15 +2,32 @@ import path from 'node:path';
 
 import { PDFNet } from '@pdftron/pdfnet-node';
 
-export async function convertToHtml(localPdfPath: string, outputHtmlPath: string, resourceBasePath: string) {
+export async function convertToHtml(
+  pdfBuffer: { type: string; data: number[] } | ArrayBuffer,
```

Comment:
why there are 2 different possible types? I think there will always be arraybuffer right?